### PR TITLE
[examples] Bump Vite.js

### PR DESCRIPTION
**Pull request by 766974616c79**
Greeting,
I updated the Vite.js example (from vite.js to version 3.0.0 and from vue to version 3.2.37) with the vite.js project creation command. 

Deployment link: [https://vite-ecru-iota.vercel.app](https://vite-ecru-iota.vercel.app)

I'll take the tips for commits messages if you have them.

#### Tests

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

File: examples/vite/README.md [hunk 1]
```diff
@@ -1,29 +1,7 @@
...
+This template should help get you started developing with Vue 3 in Vite. The template uses Vue 3 `<script setup>` SFCs, check out the [script setup docs](https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup) to learn more.
 
-While this project uses Vue.js, Vite supports many popular JS frameworks. [See all the supported frameworks](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).
+## Recommended IDE Setup
 
-## Deploy Your Own
-
-Deploy your own Vite project with Vercel.
-
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/vercel/vercel/tree/main/examples/vite&template=vite)
```
Comment (leerob) [inline examples/vite/README.md]: Can we keep these README changes?

File: examples/vite/README.md [hunk 2]
```diff
@@ -1,29 +1,7 @@
...
-```
-
-### Deploying From Your Terminal
-
-You can deploy your new Vite project with a single command from your terminal using [Vercel CLI](https://vercel.com/download):
-
-```shell
-$ vercel
-```
+- [VS Code](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar)
```
Comment (leerob) [inline examples/vite/README.md]: We can remove this.

File: examples/vite/package.json [hunk 3]
```diff
@@ -1,17 +1,18 @@
 {
   "name": "vite",
-  "version": "0.1.0",
+  "private": true,
+  "version": "0.0.0",
```
Comment (leerob) [inline examples/vite/package.json:3]: We can remove this (it's private)

**Comment by 766974616c79** [comment]
Hi,
I made the changes, and I'm sorry I didn't check the README.md 😅

**Comment by EndangeredMassa** [comment]
@MoskalykA sorry for the delay on this one. We might as well update to the latest examples at this point. If you want to do that, we can get this merged in.

**Comment by 766974616c79** [comment]
I'll do it right away

**Comment by 766974616c79** [comment]
It is good 👍🏻

**Comment by 766974616c79** [comment]
Hello,
Can anyone watch this?

**Comment by EndangeredMassa** [comment]
@MoskalykA yeah, we'll review this. Thanks for keeping up with it!

File: examples/vite/README.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
-# Vite
+# Vue 3 + Vite
 
-This directory is a brief example of a Vite/Vue.js site that can be deployed to Vercel with zero configuration.
-
-While this project uses Vue.js, Vite supports many popular JS frameworks. [See all the supported frameworks](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).
+This template should help get you started developing with Vue 3 in Vite. The template uses Vue 3 `<script setup>` SFCs, check out the [script setup docs](https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup) to learn more.
```
Comment (leerob) [inline examples/vite/README.md:3]:
```suggestion
This template should help get you started developing with Vue 3 in Vite. The template uses Vue 3 `<script setup>` SFCs, check out the [script setup docs](https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup) to learn more.

While this project uses Vue.js, Vite supports many popular JS frameworks. [See all the supported frameworks](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).
```

**Review by leerob — APPROVED** [review]

**Comment by EndangeredMassa** [comment]
I confirmed this passes CI.

Thanks @MoskalykA!

**Pull request merged**